test/button/button-subtle-fixture.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Thoughts about reusing our demo pages for the perceptual diffs if it makes sense? They'll already be bootstrapping in the typography, and we'll also likely build things like a language and RTL switcher that could come in handy.

Reply:
That's certainly possible.  We'd probably want to hide the demo container for the screenshot.  I'll look into this.